railgun/manager/run.go

```diff
@@ -112,6 +112,7 @@ func Run(ctx context.Context, c *config.Config) error {
 	}
 
 	go FlipKnativeController(mgr, proxy, &c.KnativeIngressEnabled, c, setupLog)
+	go RunHTTP(setupLog)
```

Comment:
Other additional HTTP servers we have (that are not part of controller-runtime) are started under `rootcmd.Run` and not under `manager.Run`: https://github.com/Kong/kubernetes-ingress-controller/blob/bc19341034eea5c639a49155173704ef87a4d4b7/internal/cmd/rootcmd/run.go#L10

This means that, if we want to add more HTTP servers to Railgun, we should be doing that in `rootcmd.Run`. But given the other comments (all the endpoints offered by the newly added HTTP server are either duplicates of services Railgun already offers, or raised my thoughts about their usefulness) we may well end up without that additional HTTP server.